### DISCUSSION TITLE
BM Collection compatible for TWX 9.3

### DISCUSSION THREAD

**Discussion by aumasson** [ptc-iot-sharing/BMCollectionView#51] [Q&A]
Hi Bogdan,

BMCollection is compatible for TWX 9.3 ?
I seen that the jquery is not supported in 9.3. This widget use the jquery, but perhaps, there are no impacts on the widget fonctionnalities ?

Regards,

Aurore

**Comment by BogdanMihaiciuc** [comment]
Hi Aurore,

Collection View should work just fine on 9.3, however I do recommend that you use it through [Core ui widgets](https://github.com/ptc-iot-sharing/BMCoreUIWidgets) rather than the standalone version.
